source/content/guides/drupal/drupal-unhosted/15-troubleshooting.md

```diff
@@ -1,31 +1,31 @@
 ---
-title: Migrate a Drupal 9 Site from Another Platform
+title: Migrate a Drupal Site from Another Platform
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-unhosted/troubleshooting